reactioncommerce/reaction-swag-shop
# Fixes for 1.14.0 compatibility

Resolves #69

## Fixes and changes
- Adjusting the fixtures so that the sticker's image is loaded correctly on first startup
- Created and used a new `<Link />` component on the homepage's tags block, so that the entire app is not reloaded when a featured tag is clicked
- Fixed featured product labels not displaying in grid
- Fixed the related products feature - When products are published, a "related tag" is created, that other tags can use to be marked as related to the published product
- Fixed all linting issues and removed some unused code
- Fixed sticker image not importing correctly on fresh install
- Fixed UI not scrolling to top when a similar product is clicked on PDP (and removed jQuery)
- Made footer email form only submit on enter press (not input blur)
- Linked the text in the header to the Github releases page (instead of a 404)

## Testing
1. Clone the main Reaction on this branch:`fix-dancastellon-method-hooks-async` (unless PR #4490 has been merged into 1.14.0)
2. Make sure the /private/plugins folder is empty
3. Clone this branch (`fix-ajporlante-1-14-0-compatibility`) into /imports/plugins/custom
4. Start Reaction with `METEOR_DISABLE_OPTIMISTIC_CACHING=1 SKIP_FIXTURES=1 reaction`
5. Confirm fixes/changes from above
6. Confirm overall better user experience with no bugs
